packages/mui-private-theming/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/private-theming",
-  "version": "6.4.8",
+  "version": "6.4.9",
```

Comment:
Should we manually set this to `6.4.10`? Same with `@mui/utils`

Reply:
We used to do it, there's an argument that it's not necessary. No preference, we can let code-infra decide.

Reply:
Leaving as is. Context: https://github.com/mui/material-ui/issues/45492#issuecomment-2703223155